network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -179,6 +188,7 @@ class BuyersGuidePage(RoutablePageMixin, FoundationMetadataPageMixin, Page):
         TranslatableField('hero_supporting_articles_heading'),
         SynchronizedField('cutoff_date'),
         SynchronizedField('excluded_categories'),
+        SynchronizedField('call_to_action'),
```

Comment:
**Question** Why is this synchronized rather than translatable?

Reply:
Per our discussion, tagging @TheoChevalier to see if he can help us decide if this snippet should be a SynchronizedField or a TranslatableField

Reply:
Hey, thanks for asking!

I’d say we treat this one like we treat snippets on the foundation homepage, which from a quick look are all TranslatableField

Reply:
Thanks Theo! 
I have updated this to be a [TranslatableField](https://github.com/mozilla/foundation.mozilla.org/blob/47d310b7f20ac99fa66f0c0b8a60279ae7d6ac8a/network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py#L191), as well as updated the CTA's fields to be Translatable where they can be as well, using other CTA models for reference.